setup.py

```diff
@@ -22,6 +22,10 @@
 
 
 def get_petsc_dir():
+    config = petsc4py.get_config()
+    petsc_dir = config["PETSC_DIR"]
+    petsc_arch = config["PETSC_ARCH"]
+    return (petsc_dir, path.join(petsc_dir, petsc_arch))
```

Comment:
This function is a bit redundant because `PETSC_DIR` and `PETSC_ARCH` are stored by petsc4py.